### DISCUSSION TITLE
DELETE THIS

### DISCUSSION THREAD

**Discussion by ghost** [drizzle-mizzle/CharacterAI-Discord-Bot#99] [General]
![Screenshot_20230521-062937_Discord](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/130513467/d9afb84b-c28f-4665-bf6a-2023f45735ba)

Don't trust in this, report this code

**Comment by ghost** [comment]
The bot today even started writing messages on its own off state

**Reply by justkorudev** [reply]
Well, only 1 thing:
**stop leaking your token!**

**Comment by drizzle-mizzle** [comment]
My friend, the way you got your token stolen is not anyhow related to this project...

**Reply by ghost** [reply]
I never shared my token, nor exposed it anywhere. The only one that has it is this project in its setup. I'm sorry if I'm wrong, but I really don't see the point of how someone got the token from my bot

**Reply by drizzle-mizzle** [reply]
Well, it's better for you to find it out.
I would bet on some malware browser extension.
